SharePointFramework/ProjectExtensions/src/components/TemplateSelectDialog/index.tsx

```diff
@@ -29,7 +29,7 @@ export class TemplateSelectDialog extends React.Component<
     super(props)
     this.state = {
       selectedTemplate: this._getDefaultTemplate(),
-      selectedExtensions: props.data.extensions.filter((ext) => ext.isDefault),
+      selectedExtensions: props.data.extensions.filter((ext) => this._getDefaultTemplate().listExtensionIds?.some((id) => id === ext.id)),
```

Comment:
Are we keeping the option to have a default extension? In that case, should we check for isDefault as well as check if the extension is part of the template?

Reply:
Lets keep it. This will be the possibilitiy to force a new list into all projects without "messing" with the template.